Portfolio/index.css

```diff
@@ -1,174 +1,126 @@
-console.log('Hello World');
+* {
+    h1 {
+        margin-left: 10px;
+    }
 
-body {
-    font-family: Arial, sans-serif;
-    margin: 0;
-    padding: 0;
-    background-color: #f0f0f0;
 }
-/* Additional CSS styles */
-.exp img{
-
-width: 200px;
-height: 200px;
-border: 2px solid black;
-border-radius: 100px;
 
+.navbar img {
+    margin-left: 10px;
 }
 
-/* Header styling */
-header {
-background-color: #ffffff;
-color: rgb(0, 0, 0);
-text-align: center;
-padding: 20px 0;
-}
+body {
 
-/* Section styling */
-section {
-background-color: #ffffff;
-padding: 20px;
-margin: 20px;
-border-radius: 8px;
-box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
+    background-color: #a9b5b8;
+    font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
+    color: rgb(79, 77, 77);
+    margin: 0;
+    padding: 0;
 }
 
-/* Skills section styling */
-.skills {
-display: flex;
-flex-direction: column;
-align-items: center;
+.abtme h3,

```

Comment:
The opening brace for the '.container input, textarea' selector is missing on this line.